tecken/base/symboldownloader.py

```diff
@@ -31,33 +29,6 @@ def __init__(self, status_code, url):
         self.url = url
 
 
-def iter_lines(stream, chunk_size=ITER_CHUNK_SIZE):
```

Comment:
This was something used by the symbolication code ages ago and is no longer used, so I nixed it.